bindings/pyroot/pythonizations/test/numbadeclare.py

```diff
@@ -128,6 +128,23 @@ def fn13(x):
         self.assertEqual(mean_x.GetValue(), 1.5)
         self.assertEqual(mean_y.GetValue(), 3.0)
 
+    @unittest.skipIf(skip, skip_reason)
+    def test_rdataframe_temporary(self):
+        """
+        Test passing a temporary from an RDataFrame operation
+        """
+        @ROOT.Numba.Declare(["const RVecF"], "RVecF")
```

Comment:
passing a temporary does not seem to work if the parameter is just `RVecF` (as opposed to `const RVecF`). Is that intentional?

Reply:
Yes, since it's a temporary, without the `const` cling tells me:
```
input_line_43:2:74: error: non-const lvalue reference to type 'RVec<...>' cannot bind to a temporary of type 'RVec<...>'
auto func1(ROOT::VecOps::RVec<float>& var0){return Numba::pass_temporary(var0[var0 > 0])
                                                                         ^~~~~~~~~~~~~~
input_line_32:5:41: note: passing argument to parameter 'x_0' here
ROOT::RVecF pass_temporary(ROOT::RVecF& x_0) {
```

Reply:
Note also that multiple decorators can be stack to define multiple overloads, e.g.:
```python
@ROOT.Numba.Declare(["RVecF"], "RVecF")
@ROOT.Numba.Declare(["const RVecF"], "RVecF")
def pass_temporary(v):
    return v*np.array([2.]).astype(np.float32)
```